src/Microsoft.VisualStudio.Services.Agent/Blob/DedupManifestArtifactClientFactory.cs

```diff
@@ -203,6 +209,15 @@ private DedupManifestArtifactClientFactory()
                 maxParallelism = DefaultDedupStoreClientMaxParallelism;
             }
             traceOutput($"Max dedup parallelism: {maxParallelism}");
+
+            ClientSettingsInfo clientSettings = clientType is null
+                ? null
```

Comment:
Is there not a client settings for default/unknown clienttype?

Reply:
there's no https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps?path=/BlobStore/Client/WebApi/Contracts/ClientSettingsInfo.cs&version=GBmaster&line=22&lineEnd=23&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents